eclipse-collections-code-generator-maven-plugin/src/main/java/org/eclipse/collections/codegenerator/maven/GenerateMojo.java

```diff
@@ -31,12 +31,12 @@
     /**
      * Skips code generation if true.
      *
-     * @parameter expression="${skipCodeGen}"
+     * @parameter property="skipCodeGen"
      */
     private boolean skipCodeGen;
 
     /**
-     * @parameter expression="${project.build.directory}/generated-sources"
+     * @parameter default-value="${project.build.directory}/generated-sources"
```

Comment:
Per [this API doc](https://maven.apache.org/developers/mojo-api-specification.html), think this should look like below? Guess original `expression` should have been `default-value` to begin with. CC @motlin for confirmation. 

```
@parameter property="templateDirectory" default-value="${project.build.directory}/generated-sources"
```